build/LegacyPackages/14.0/Microsoft.Build.Tasks.Core/Microsoft.Build.Tasks.Core.cs

```diff
@@ -903,8 +903,11 @@ public sealed partial class ResolveCodeAnalysisRuleSet : Microsoft.Build.Tasks.T
         public string ResolvedCodeAnalysisRuleSet { get { throw null; } }
         public override bool Execute() { throw null; }
     }
-#if !NETSTANDARD
+#if NETSTANDARD
```

Comment:
I thought the `LegacyPackages` folder was dead. Did something force you to do this, or were you being proactive? If the latter, probably just revert it -- we'll clean up later.

Reply:
Proactive 😄 Will revert and see what happens.